client/components/mma/accountoverview/PersonalisedHeader.tsx

```diff
@@ -1,12 +1,26 @@
 import { css } from '@emotion/react';
 import { headline, space } from '@guardian/source-foundations';
+import { dateString } from '../../../../shared/dates';
 import type { MembersDataApiResponse } from '../../../../shared/productResponse';
 import { isProduct, sortByJoinDate } from '../../../../shared/productResponse';
 
 interface PersonalisedHeaderProps {
 	mdapiResponse: MembersDataApiResponse;
 }
 
+function calculateTimeOfDay() {
```

Comment:
Should we leave this as an arrow function for consistency? (I realise we're not _entirely_ consistent with this elsewhere…)

Reply:
When I was pairing with @KentGrm he pointed out that if you define the function as an arrow function inside the component, every time the component is re-rendered the function will be defined again

Reply:
See also this SO answer. I don't think it's a big performance hit at all but might be a better practice going forward. https://stackoverflow.com/a/46138370

Reply:
useCallback is useful if you want to access variables in the render/component function. But this is a code smell and edgecase

Reply:
Yeah not flagging the useCallback part of that answer just this part:

> In most cases you should declare the functions outside the component function so you declare them only once and always reuse the same reference. When you declare the function inside, every time the component is rendered the function will be defined again.

Reply:
Ah, that's an interesting point about functions being defined every time a component is rendered. In this case the function is no longer inside a component so it doesn't matter if it's an arrow function? (But it's fine either way!) This is an unexpected, but interesting, chat in a PR that's ostensibly about copy changes 😀